pkg/cri/resource-manager/control/page-migrate/demoter.go

```diff
@@ -97,32 +98,69 @@ func copyPagePool(p pagePool) pagePool {
 	return c
 }
 
-func (d *demoter) UpdateDemoter(cid string, p pagePool, targetNodes system.IDSet) {
+func createDemoter(m *migration) *demoter {
+	return &demoter{
+		migration:         m,
+		containerDemoters: make(map[string]chan interface{}, 0),
+		pageMover:         &linuxPageMover{},
+	}
+}
+
+func (d *demoter) start() {
+	if d.pageScanInterval > 0 && d.pageMoveInterval > 0 && d.maxPageMoveCount > 0 {
+		log.Info("scanning pages every %s, moving max. %d pages every %s",
+			d.pageScanInterval.String(), d.maxPageMoveCount, d.pageMoveInterval.String())
+		d.startDirtyBitResetTimer()
+	} else {
+		log.Info("scanning pages is disabled")
+	}
+}
+
+func (d *demoter) Stop() {
```

Comment:
nit: I'd add comment to the exported methods. Or, alternatively, if demoter implements an interface state that in the demoter definition

Reply:
Added docstring to exported methods Stop() and Reconfigure().